scripts/deploy-live-comms.sh

```diff
@@ -0,0 +1,32 @@
+
+#!/bin/sh
+
```

Comment:
Add documentation for this script either in scripts/readme or move it to live comms (?)